docs: update tasks section (#889)
apify/apify-docs: sources/platform/actors/running/tasks.md

sidebar_position: 7.2
slug: /actors/running/tasks
---

**Create and save reusable configurations of Apify Actors tailored to specific use cases.**

---

Actor tasks let you create multiple reusable configurations of a single Actor, adapted for specific use cases. For example, you can create one **Web Scraper** ([apify/web-scraper](https://apify.com/apify/web-scraper)) configuration (task) that scrapes the latest reviews from imdb.com, another that scrapes nike.com for the latest sneakers, and a third that scrapes your competitor's e-shop. You can then use and reuse these configurations directly from [Apify Console](https://console.apify.com/actors/tasks), [Schedules](../../schedules.md), or [API](/api/v2#/reference/actor-tasks/run-collection/run-task).

You can find all your tasks at [https://console.apify.com/actors/tasks](https://console.apify.com/actors/tasks).

## Create

To create a task, open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create task** button.

![Create a new Apify task](./images/tasks/create-task.png)

## Configure

> You can set up your task's input under the **Input and options** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.

Reviewer: It's just Input now
```suggestion
You can set up your task's input under the **Input** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.
```

![Apify task configuration](./images/tasks/create-task-configure.png)

An Actors' input fields may vary depending on their purpose, but they all follow the same principle: _you provide an Actor with the information it needs so it can do what you want it to do._

You can set run options such as timeout and [memory](./usage_and_resources.md) at the bottom of the input section.

> ### Identification

Reviewer: I personally prefer the old title here - sure it's longer, but I think it's clearer

Author: Here I tried to make sure that all the headings are in the similar style while still conveying the same information as. before

Reviewer: I see what you mean, but the old version just reads easier. Really, though, shouldn't this section just be called `Naming`?

Author: Yup that works, my intention here was to keep the parallel structure of headings, but `Naming` definitely achieves the same result

To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least 3 characters long with 63 characters limit.

### Settings

> You can grant [access rights](../../collaboration/index.md) to other Apify users under the **Settings** tab of the task's detail page.

Reviewer: Sharing is now under this button

![CleanShot 2024-03-21 at 11 33 55@2x](https://github.com/apify/apify-docs/assets/38246758/ec8e96dc-2c7c-4606-9109-6ea22d5051b2)

## Run

> Once you've configured your task, you can run it using the **Run** button in the bottom-left side of the **Input and options** tab.

Reviewer: I guess the move of the Run button is going to mess some things up for Docs

```suggestion
Once you've configured your task, you can run it using the **Run** button in the top-right side of the screen.
```

You can find all your tasks in the [Apify Console](https://console.apify.com/actors/tasks).

## Create

To create a task, open any Actor from [Apify Store](https://console.apify.com/store) or your list of [Actors](https://console.apify.com/actors) in Apify Console. At the top-right section of the page, click the **Create task** button.

![Create a new Apify task](./images/tasks/tasks-create-task.png)

## Configure

You can set up your task's input under the **Input** tab. A task's input configuration works just like an Actor's. After all, it's just a copy of an Actor you can pre-configure for a specific scenario. You can use either JSON or the visual input UI.

![Apify task configuration](./images/tasks/tasks-create-configure.png)

An Actors' input fields may vary depending on their purpose, but they all follow the same principle: _you provide an Actor with the information it needs so it can do what you want it to do._

You can set run options such as timeout and [memory](./usage_and_resources.md) in the **Run options** tab of the task's input configuration.

### Naming

> To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least `3` characters long with `63` characters limit.

Reviewer: ```suggestion
To make a task easier to identify, you can give it a name, title, and description by clicking its caption on the detail page. A task's name should be at least `3` characters long with a limit of `63` characters.
```

## Run

> Once you've configured your task, you can run it using the **Start** button in the top-right side of the screen.

Reviewer: ```suggestion
Once you've configured your task, you can run it using the **Start** button on the top-right side of the screen.
```

- Directly via the [Apify API](/api/v2#/reference/actor-tasks/run-collection/run-task).
- The [JavaScript API client](/api/client/js/reference/class/TaskClient).
- The [Python API client](/api/client/python/reference/class/TaskClient).

> ## Share

Reviewer: This is a duplicate of the `Settings` - I guess Settings can be deleted?
